### ISSUE TITLE
Version conflict when installing via PIP

### ISSUE THREAD

**Issue by shogunbr** [collective/transmogrify.dexterity#47]
transmogrify.dexterity 2.0.0 is not installable via pip due ==2.7 constraint in python_requires
https://github.com/collective/transmogrify.dexterity/blob/42689f09006b37042d6de5ff219a3e6d7c20c548/setup.py#L55

**Comment by wesleybl** [comment]
Yes this is a bug. Should be:

```python
python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*!=3.5.*",
```

**Comment by wesleybl** [comment]
Fix in #48 48
